Android/app/src/main/java/io/github/project_travel_mate/travel/Hotels.java

```diff
@@ -330,22 +373,25 @@ public void onClick(View view) {
                 e.printStackTrace();
                 Log.e("ERROR : ", "Message : " + e.getMessage());
             }
-            return vi;
+            return convertView;
         }
-    }
 
-    public void onClick(View view) {
-        switch (view.getId()) {
-            case R.id.city :
-                Intent i = new Intent(Hotels.this, SelectCityFragment.class);
-                startActivity(i);
-                break;
-            case R.id.seldate :
-                mDatePickerDialog.setVibrate(isVibrate());
-                mDatePickerDialog.setYearRange(1985, 2028);
-                mDatePickerDialog.setCloseOnSingleTapDay(isCloseOnSingleTapDay());
-                mDatePickerDialog.show(getSupportFragmentManager(), DATEPICKER_TAG);
-                break;
+        class ViewHolder {
+            @BindView(R.id.VideoTitle)
+            TextView title;
+            @BindView(R.id.VideoDescription)
```

Comment:
Change the id name to `hotel_address`.
Change it here: https://github.com/project-travel-mate/Travel-Mate/blob/master/Android/app/src/main/res/layout/hotel_listitem.xml#L62